lua/entities/gmod_tardis/modules/sh_teleport.lua

```diff
@@ -111,9 +112,11 @@ if SERVER then
 						end
 						self:SetData("prevortex-flight",nil)
 						self:SetSolid(SOLID_VPHYSICS)
-						self.phys:EnableMotion(true)
-						self.phys:Wake()
-						
+						if not self:GetData("physlock",false) then
```

Comment:
This should be handled via a hook, call some kind of TeleportComplete hook if not done already and handle in the physlock module